src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/config/SwaggerConfiguration.java

```diff
@@ -1,52 +1,40 @@
 package uk.gov.hmcts.reform.wataskmanagementapi.config;
 
+import io.swagger.v3.oas.annotations.enums.ParameterIn;
+import io.swagger.v3.oas.models.Operation;
+import io.swagger.v3.oas.models.media.StringSchema;
+import io.swagger.v3.oas.models.parameters.Parameter;
+import org.springdoc.core.GroupedOpenApi;
+import org.springdoc.core.customizers.OperationCustomizer;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.web.bind.annotation.RestController;
-import springfox.documentation.builders.PathSelectors;
-import springfox.documentation.builders.RequestHandlerSelectors;
-import springfox.documentation.builders.RequestParameterBuilder;
-import springfox.documentation.service.ParameterType;
-import springfox.documentation.service.RequestParameter;
-import springfox.documentation.spi.DocumentationType;
-import springfox.documentation.spring.web.plugins.Docket;
-import springfox.documentation.swagger2.annotations.EnableSwagger2;
-
-import static java.util.Arrays.asList;
+import org.springframework.web.method.HandlerMethod;
 
 @Configuration
-@EnableSwagger2
 public class SwaggerConfiguration {
```

Comment:
Can we rename this file to be consistent with the updated Case Event Handler please - `OpenAPIConfiguration`

Reply:
File renamed to OpenAPIConfiguration.java